documentation/devon4j-layers.asciidoc

```diff
@@ -250,52 +244,48 @@ public class BookingmanagementImpl extends AbstractComponentFacade implements Bo
 }
 ----
 
-In the above _My Thai Star_ logic layer example we can see:
+In the above _MyThaiStar_ logic layer example we can see:
 
-- business logic and/or https://github.com/devonfw/devon4j/wiki/guide-beanmapping[object mappings].
+- Business logic and/or https://github.com/devonfw/devon4j/wiki/guide-beanmapping[object mappings].
 
 - Delegation of the transaction management through the Spring's `@Transactional` annotation.
 
 === Transfer Objects
 In the code examples of the _logic_ layer section you may have seen a _BookingCto_ object. This is one of the https://github.com/devonfw/devon4j/wiki/guide-transferobject[transfer objects] defined in _devon4j_ to be used as transfer data element between layers.
 
-Main benefits of using _TO's_:
+Main benefits of using _Transfer Objects_:
 
 - Avoid inconsistent data (when entities are sent across the app changes tend to take place in multiple places).
 
 - Define how much data to transfer (relations lead to transferring too much data).
 
 - Hide internal details.
 
-In _devon4j_ we can find two different _Transfer Objects:
+In _devon4j_ we can find two different _Transfer Objects_:
```

Comment:
Are there not three TO? Or is the 'BusinessObject' not part of this as well? At least the guide-pdf says so.

Reply:
I guess you're right, but the documentation only mentions BusinessObject's in the context of the naming scheme, i.e.: `«BusinessObject»Eto` and `«BusinessObject»«Subset»Cto`. There's also a "Service Transfer Object" mentioned in the guide (which is supposed to follow the naming scheme `«BusinessObject»Sto`) which isn't referenced here either ...

@sjimenez77 do you have any input on this?